_episodes/03-monitor-the-model.md

```diff
@@ -198,9 +198,23 @@ Setting the `random_state` to `0` is a short-hand at this point. Note however, t
 
 In episode 2 we trained a dense neural network on a *classification task*. For this one hot encoding was used together with a `Categorical Crossentropy` loss function.
 This measured how close the distribution of the neural network outputs corresponds to the distribution of the three values in the one hot encoding.
-Now we want to work on a *regression task*, thus not predicting a class label (or integer number) for a datapoint. In regression, we like to predict one (and sometimes many) values of a feature. This is typically a floating point number. 
+Now we want to work on a *regression task*, thus not predicting a class label (or integer number) for a datapoint. In regression, we like to predict one (and sometimes many) values of a feature. This is typically a floating point number.
 
-In our example we want to predict the sunshine hours in Basel (or any other place in the dataset) for tomorrow based on the weather data of all 18 locations today. `BASEL_sunshine` is a floating point value (i.e. `float64`). The network should hence output a single float value which is why the last layer of our network will only consist of a single node. 
+> ## Exercise: Architecture of the network
+> As we want to design a neural network architecture for a regression task,
+> see if you can first come up with the answers to the following questions:
+> 1. What must be the dimension of our input layer?
+> 2. How would our output layer look like? What about the activation function? Tip: Remember that the activation function in our previous classification network scaled the outputs between 0 and 1.
```

Comment:
The idea is to let the learners think about this a bit before giving them the solution on how to do this directly. But you're right that it's a tricky question, especially the activation function. Would you have suggestions on how to rephrase the question so that we still trigger this thought process, but without making it too difficult?

Reply:
That is a good question. the challenge is, that `softmax` involves quite some math from the point of view of a learner. Here, we could argue, that this math is simply not needed first. This way learners are reassured that they need to consider something else. Then, the concept of the activation function transforming the outputs is important as it has direct consequences onto the variable being regressed.

How about we add a concrete hint:
```suggestion
> 2. We want to to output the prediction of a single number. The output layer of the NN hence cannot be the same as for the classification task earlier. This is because the `softmax` activation being used had a concrete meaning with respect to the class labels which is not needed here. What output layer design would you choose for regression?  
Hint: A layer with `relu` activation, with `sigmoid` activation or no activation at all? 
```